typings/index.d.ts

```diff
@@ -295,7 +295,7 @@ export class AerospikeRecord<B extends AerospikeBins = AerospikeBins> {
      *
      * @type {AerospikeBins}
      */
-    public bins: B;
+    public bins: AerospikeRecord<B>;
```

Comment:
This change was moved incorrectly. It was originally made for `BatchResult` class and `.record` field, but now it affects `AerospikeRecord` class, and for now, it's impossible to access record's bins, since there is loop reference to AerospikeRecord type, instead of bins type (`B` generic)